whatsappcrm_backend/cbz_integration/views.py

```
@@ -1594,6 +1595,310 @@ def cbz_certificate_renew_view(request: HttpRequest) -> JsonResponse:
+        logger.warning(
+            "CBZ_3DS_RETURN_URL not configured — iVeri cannot POST 3DS result back. "
+            "Set CBZ_3DS_RETURN_URL to the public URL of /crm-api/payments/cbz/card/3ds/return/"
+        )
+        return JsonResponse(
+            {"success": False, "message": "3DS return URL not configured. Contact support."},
+            status=503,
+        )
+
+    try:
+        booking = _resolve_or_create_booking(payload, amount)
+    except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)